doc/hyperfine.1

```diff
@@ -207,6 +214,14 @@ Export the results of a parameter scan benchmark to a markdown table:
 \fBhyperfine\fR \fB\-\-export\-markdown\fR output.md \fB\-\-parameter-scan\fR time 1 5 'sleep {time}'
 .fi
 .RE
+.LP
+Demonstrate when each of \fB\-\-setup\fR, \fB\-\-prepare\fR, \fIcmd\fP and \fB\-\-cleanup\fR will run:
+.RS
+.nf
+\fBhyperfine\fR \fB\-\fRL n 1,2 \fB\-r\fR 2 \fB\-\-show-output\fR \fB\-s\fR 'echo setup n={n}' \fB\-\fRp 'echo warmup n={n}' 'echo test n={n}' \fB\-\fRc 'echo clean n={n}'
```

Comment:
Can we please use the long version of the command line options here? Not everyone is familiar with all the shortcuts and the long options have a self-documenting character. The command could be split over multiple lines using `\`.